packages/rum-core/src/boot/startRum.spec.ts

```diff
@@ -292,47 +265,90 @@ describe('rum events url', () => {
     expect(serverRumEvents[1].view.url).toEqual('http://foo.com/bar')
   })
 
-  it('should keep the same URL when updating an ended view', () => {
-    const { notifyPerformanceEntries } = mockPerformanceObserver()
-    const { clock, changeLocation } = setupBuilder.withFakeClock().withFakeLocation('http://foo.com/').build()
+  describe('when clock ticks', () => {
```

Comment:
**🥜 nitpick:** ‏instead of grouping those tests because they need the mocked clock, maybe mock the clock in tests individually, example:

```js
 it('should attach the url corresponding to the start of the event', () => {
      const clock = mockClock()
      registerCleanupTask(clock.cleanup)
      setupViewUrlTest()
      clock.tick(10)
```

This is a bit more verbose for now, but in a future PR we should do the `registerCleanupTask` directly in `mockClock`, which would mitigate the verbosity.